server/api/init/init.ts

```diff
@@ -0,0 +1,39 @@
+import { Express, Request, Response } from 'express'
+import { ApiEndPoint } from '@common/api/endpoint'
+import { sendErr } from '@server/utils/requests'
+import {
+  AssessmentService,
+  AssessmentCountryService,
+  AssessmentRegionService,
+  RegionGroupService,
+} from '@server/service'
+
+export const InitGet = {
+  init: (express: Express): void => {
+    express.get(ApiEndPoint.Init.one(), async (req: Request, res: Response) => {
+      const assessmentName = req.params.name ?? 'fra'
```

Comment:
no fra hardcorded. read it from settings table if assessment name is not in the request